boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuflpr.dts

```diff
@@ -21,6 +21,21 @@
 	};
 };
 
+&cpuflpr_vevif_rx {
+	interrupts = <16 NRF_DEFAULT_IRQ_PRIORITY>,
+				<17 NRF_DEFAULT_IRQ_PRIORITY>,
+				<18 NRF_DEFAULT_IRQ_PRIORITY>,
+				<19 NRF_DEFAULT_IRQ_PRIORITY>,
+				<20 NRF_DEFAULT_IRQ_PRIORITY>,
+				<21 NRF_DEFAULT_IRQ_PRIORITY>,
+				<22 NRF_DEFAULT_IRQ_PRIORITY>;
+	nordic,tasks-mask = <0x007f0000>;
+};
+
+&cpuflpr_vevif_tx {
+	nordic,events-mask = <0x00020000>;
+};
```

Comment:
why is board modifying this?

Reply:
should be ok now

Reply:
I do not see it in https://github.com/zephyrproject-rtos/zephyr/pull/77341/files#diff-8e93c65c9c86c18f332d6de70f274bda09e6a0fea3110153c93d0bfb251fbad8

Reply:
ping, I still see this in board files. question is why soc level config is kept in a board file?

Reply:
done